src/transformStateWithClones.js

```diff
@@ -8,6 +8,24 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  let stateClone = { ...state };
+  const stateHistory = [];
+
+  for (const action of actions) {
```

Comment:
Use destruction for `action`
```suggestion
  for (const action of actions) {
    const { type, ......... } = action;
  
```

Reply:
Still not fixed from the previous review 
<img width="919" alt="image" src="https://github.com/mate-academy/js_stateful-clones/assets/64750958/f27bff87-1180-4e7c-b45d-3de814149f08">